### ISSUE TITLE
tf2.3 not find "keras.optimizers.schedules.CosineDecay"

### ISSUE THREAD

**Issue by zuixiaosanlang** [FLming/CRNN.tf2#25]

**Comment by FLming** [comment]
Sorry, it was my mistake. `CosineDecay` was imported in version 2.5, the previous version can be replaced by other methods to control the learning rate.
